build_tools/travis/test_script.sh

```diff
@@ -45,6 +57,7 @@ run_tests() {
 
     set -x  # print executed commands to the terminal
 
+    pytest_collects_same_tests_in_three_runs sklearn
```

Comment:
On average each collect run takes ~6 seconds, so three runs ~18 seconds. If we keep this test, it would be good to run it on just one of the CI setups.

Since we are moving to azure, it would be good to test if the parallel tests would help on azure. If they do help, this collect test could be removed.